.github/CODEOWNERS

```diff
@@ -2,13 +2,13 @@
 * @myungjoo @jijoongmoon @again4you @jaeyun-jung
 
 # C-API
-/c/ @jaeyun-jung @again4you @myungjoo @helloahn @anyj0527 @gichan-jang
+/c/ @jaeyun-jung @again4you @myungjoo @anyj0527 @gichan-jang @songgot
 
 # C++-API (SNAP & nntrainer, nnstreamer-pipeline)
-/cpp/ @nnstreamer/snap @jijoongmoon @kparichay @myungjoo @again4you
+/cpp/ @nnstreamer/snap @jijoongmoon @myungjoo @again4you
```

Comment:
github makes error about this line `@nnstreamer/snap`. Please check the permission.
```
Unknown owner on line 8: make sure the team @nnstreamer/snap exists and has write access to the repository
/cpp/ @nnstreamer/snap @jijoongmoon @myungjoo @again4you
```

Reply:
granted write permission to snap group.